packages/test/src/test-integration.ts

```diff
@@ -349,6 +349,42 @@ if (RUN_INTEGRATION_TESTS) {
     t.is(timerEvents[3].timerCanceledEventAttributes!.timerId, '0');
   });
 
+  test('patched', async (t) => {
+    const client = new WorkflowClient();
+    const workflow = client.stub<Empty>('patched', { taskQueue: 'test' });
+    const runId = await workflow.start();
+    const res = await workflow.result();
+    t.is(res, undefined);
+    const execution = await client.service.getWorkflowExecutionHistory({
+      namespace,
+      execution: { workflowId: workflow.workflowId, runId },
+    });
+    const hasChangeEvents = execution.history!.events!.filter(
+      ({ eventType }) => eventType === iface.temporal.api.enums.v1.EventType.EVENT_TYPE_MARKER_RECORDED
+    );
+    // There will only be one marker despite there being 2 hasChange calls because they have the
```

Comment:
I dunno that I agree here. It's easy enough to rely on Core to do the right thing here and not have to think about it in lang, and there's no overhead besides a few extra bytes since this command isn't awaited, so the number of calls to core remains the same.

If this changes later, saying we just always send commands means there are less things to change lang side.